OpenImis.ModulesV2/InsureeModule/Models/EnrollFamilyModels/Enrolment.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenImis.ModulesV2.InsureeModule.Models.EnrollFamilyModels
+{
+    public class Enrolment
```

Comment:
We need to create a ticket to improve the Family model to follow the model from the specification document, section 2.1.1.

Reply:
Do you think we should follow the same logic as GetFamily from the begining? We will anyway need to change the mobile apps for GetFamily.

Reply:
JSON proposal:

```
{
  "Family": 
  [
    {
      "FamilyId": "integer",
      "InsureeId": "integer",
      "LocationId": "integer",
      "HOFCHFID": "string",
      "Poverty": "boolean",
      "FamilyType": "boolean",
      "FamilyAddress": "string",
      "Ethnicity": "string",
      "ConfirmationNo": "string",
      "ConfirmationType": "string",
      "isOffline": "boolean",
      "Insurees": 
      [
        {
	  "InsureeId": "integer",
          "FamilyId": "integer",
          "CHFID": "string",
          "LastName": "string",
          "OtherNames": "string",
          "DOB": "yyyy-mm-dd",
          "Gender": "int",
          "Marital": "int",
          "isHead": "boolean",
          "IdentificationNumber": "string",
          "Phone": "string",
          "PhotoPath": "string",
          "CardIssued": "boolean",
          "Relationship": "int",
          "Profession": "int",
          "Education": "int",
          "Email": "string",
          "TypeOfId": "int",
          "HFID": "int",
          "CurrentAddress": "string",
          "GeoLocation": "string",
          "CurVillage": "int",
          "isOffline": "boolean",
          "EffectiveDate": "yyyy-mm-dd",
          "Picture":
            {
              "ImageName":"string",
              "ImageContent":"byteArray"
            }
        }
      ],
      "Policies": 
      [
        {
          "PolicyId": "integer",
          "FamilyId": "integer",
          "EnrollDate": "yyyy-mm-dd",
          "StartDate": "yyyy-mm-dd",
          "EffectiveDate": "yyyy-mm-dd",
          "ExpiryDate": "yyyy-mm-dd",
          "PolicyStatus": "integer",
          "PolicyValue": "decimal",
          "ProdId": "integer",
          "OfficerId": "integer",
          "PolicyStage": "integer",
          "isOffline": "boolean",
          "Premium": 
            {
              "PremiumId": "integer",
              "PolicyId": "integer",
              "PayerId": "integer",
              "Amount": "string",
              "Receipt": "string",
              "PayDate": "yyyy-mm-dd",
              "PayType": "integer",
              "isPhotoFee": "boolean",
              "isOffline": "boolean"
            }
        }
      ],
      "InsureePolicy": 
      [
        {
          "InsureeId": "integer",
          "PolicyId": "integer",
          "EffectiveDate": "yyyy-mm-dd"
        }
      ]
    }
  ]
}
```
DIAGRAM from SSMS
![Family-diagram](https://user-images.githubusercontent.com/23075165/63346721-e863c580-c355-11e9-8d5a-2b00b9aa1521.jpg)

Reply:
Ok for the proposal.